vcpkg/commands/format-manifest.md

```diff
@@ -0,0 +1,54 @@
+---
+title: vcpkg format-manifest
+description: Reference for the vcpkg format-manifest command. Formats vcpkg.json files and converts CONTROL files to vcpkg.json format.
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 08/17/2023
+ms.prod: vcpkg
+---
+
+# vcpkg format-manifest
+
+## Synopsis
+
+```console
+vcpkg format-manifest [--all] [--convert-control] [<path>...]
+```
+
+## Description
+Formats manifest (`vcpkg.json`) files and converts control (`CONTROL`) files into the manifest format.
+
+>[!NOTE]
+>If both `CONTROL` and `vcpkg.json` file are present in the same directory, the command will exit with an error.
```

Comment:
I think this note isn't important enough to alert the user.

Reply:
It is important to document the behavior, but not important enough that we have to call it out with `[!NOTE]`.